tests/test_deployments/test_app.py

```diff
@@ -221,6 +221,10 @@ def test_handle_notfound(self) -> None:
             response = client.get("/non_existing_endpoint")
             assert response.status_code == 404
 
+    @pytest.mark.skipif(
+        platform.system() == "Windows",
+        reason="this endpoint makes request to the local tendermint node using address 0.0.0.0 which does not work on windows",
+    )
```

Comment:
This will require a change on the `TendermintNode` class implementation, so till then we skip this. https://github.com/valory-xyz/open-autonomy/issues/977